app/controllers/users/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Users::PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
+
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.posts.build
+  end
+
+  def show
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:id])
+  end
+end
```

Comment:
- Good job done so far, but there is no need for this controller, I suggest you remove this and set up your controllers properly 🙏